Scripts/new_feature.sh

```
@@ -107,10 +107,16 @@ final class ${FEATURE_NAME}Tests: XCTestCase {
 }
 EOF
 
-# Add to app dependencies
+# Convert to lowerCamelCase
 LOWER_CAMEL_CASE_FEATURE_NAME="$(tr '[:upper:]' '[:lower:]' <<< "${FEATURE_NAME:0:1}")${FEATURE_NAME:1}"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, here-strings are undefined.

(SC3011)